code/modules/busy_space/organizations.dm

```diff
@@ -278,7 +346,37 @@
 	headquarters = "Luna, Sol"
 	motto = ""
 
-	ship_prefixes = list("HTV" = "freight", "HLV" = "munitions resupply", "HDV" = "asset protection", "HDV" = "preemptive deployment")
+	missions = list(
+		new/datum/lore/mission/prebuilt/defense("HDV"),
+		new/datum/lore/mission/prebuilt/industrial("HIV"),//extremely unfortunate prefix. can probably fudge it as HFV or HLV if this is too unpleasant
```

Comment:
Yeah might as well switch it to HLV just for the sake of avoiding any weirdness.

Reply:
done